app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/settings/PrivacyOptionsView.java

```diff
@@ -323,6 +320,9 @@ private void setWebXR(boolean value, boolean doApply) {
 
         if (doApply) {
             SettingsStore.getInstance(getContext()).setWebXREnabled(value);
+            for (WindowWidget window: mWidgetManager.getWindows().getCurrentWindows()) {
+                window.getSession().reload(GeckoSession.LOAD_FLAGS_NONE);
```

Comment:
Should probably be using `GeckoSession.LOAD_FLAGS_BYPASS_CACHE` as it has been fixed in GV.